docs/fsharp/style-guide/formatting.md

```diff
@@ -1708,6 +1897,25 @@ let myNumber =
 The domain that's being modeled should ultimately drive the naming convention.
 If it's idiomatic to use a different convention, that convention should be used instead.
 
+If the return value of an expression is a computation expression, prefer putting the computation expression keyword name on its own line
+
+```fsharp
+// ✔️ OK
+let foo () = 
+    async {
+        let! value = getValue()
+        do! somethingElse()
+        return! anotherOperation value 
+    }
+    
+// ❌ Not OK, but formatters may allow this if preferred // TODO Should these just both be acceptable styles?
```

Comment:
Yeah, I guess both styles are ok. I'm not sure how to phrase this. You want to add something that people shouldn't mix the two styles in teams.